assets/src/edit-story/components/colorPicker/currentColorPicker.js

```diff
@@ -173,6 +242,16 @@ function CurrentColorPicker({ rgb, hsl, hsv, hex, onChange, showOpacity }) {
       </Body>
       <Footer>
         <HexValue>
+          <Button
+            variant={BUTTON_VARIANTS.SQUARE}
+            type={BUTTON_TYPES.QUATERNARY}
+            size={BUTTON_SIZES.SMALL}
+            aria-label={__('Pick a color from canvas', 'web-stories')}
+            onClick={initEyedropper}
+          >
+            <Icons.Eyedropper style={{ padding: 6 }} />
```

Comment:
The icons should already be imported so that they have 32x32 size and padding is included (see the other icons / storybook).

Reply:
I was looking for it, found it now under "Pipette" 😃